tests/conftest.py

```diff
@@ -17,3 +37,47 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
+
+
+@pytest.fixture(scope="session")
+def model_spec_path() -> Path:
+    repo_path = paths.BASE_DIR
+    config_path = repo_path / "model_specifications" / "model_spec.yaml"
+    return config_path
+
+
+@pytest.fixture(scope="session")
+def simulation_states(model_spec_path: Path) -> dict[str, InteractiveContext]:
+    # Set up interactive sim
+    sim = InteractiveContext(model_spec_path)
+    sim_states = {}
+    for step in SIMULATION_STEPS:
+        sim_states[step] = sim
+        # The simulation starts on the pregnancy time step
+        if step == SIMULATION_EVENT_NAMES.LATE_NEONATAL_MORTALITY:
+            continue
+        sim.step()
+
+    return sim_states
```

Comment:
Does this actually do what you want it to do? It looks to me like this just creates a list containing pointers to the same simulation object at each index.

Reply:
I've updated how I'm access the InteractiveContext since I couldn't get the backup to work in the pytest.